language/golang/language.go

```diff
@@ -99,9 +99,10 @@ func (l *Language) Execute(logger *log.Logger, repositoryPath string) (coverage
 		Directory: repositoryPath,
 	})
 	if err != nil {
-		return 0, problems, pkgerrors.WithMessage(pkgerrors.WithStack(err), commandOutput)
+		return nil, nil, pkgerrors.WithMessage(pkgerrors.WithStack(err), commandOutput)
```

Comment:
I don't like that very much. It makes debugging and refactorings harder, because one has to analyze if "problems" can actually be non-nil. This is especially annoying for refactorings.